devito/operator/operator.py

```diff
@@ -495,7 +495,11 @@ def _prepare_arguments(self, autotune=None, **kwargs):
                 # E.g., SubFunctions
                 continue
             for k, v in p._arg_values(**kwargs).items():
-                if k in args and args[k] != v:
+                try:
```

Comment:
wait... what's this one for?

Reply:
The cordinates end up here in some sparse reduces case. Not sure exactly why.